plugin.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Plugin Name: Altis Browser Security
+ * Description: Browser security utilities for WordPress/Altis
+ * Author: Human Made
+ * Author URI: https://humanmade.com/
+ */
+
+namespace Altis\Security\Browser;
+
+if ( function_exists( 'Altis\\Security\\bootstrap' ) ) {
+	// Being loaded via Altis, skip bootstrap.
+	return;
+}
+
+bootstrap( [
```

Comment:
Is the assumption this plugin can & will only be installed via composer? Why not just `require_once` in here after the function exists check? Can also drop that check altogether as we don't need to include this file at all in Altis.

Reply:
Hmm, true. I did want to keep it there just in case it ends up installed in the wrong spot.